### PR TITLE
Requested by Iteq - Adds Deathstings to Changeling

### PR DESCRIPTION
**Pull request by silicons**
:^)

`btw this is a meme please dont merge`

### CODE REVIEW

**Comment by coiax** [comment]
Disappointed that this doesn't call death() on the mob.

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 1]
```diff
@@ -779,3 +779,13 @@
...
+
+/datum/reagent/toxin/deathsting
+	name = "Death Oil"
+	id = "deathoil"
+	description = "An extremely potent toxin that causes death unless prompt medical attention is given to the victim."
+	toxpwr = 2.5
+	metabolization_rate = 4 * REAGENTS_METABOLISM
+
+/datum/reagent/toxin/deathsting/on_mob_life(mob/living/M)
+	M.adjustOxyLoss(10)
```
Comment (ChangelingRain) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: M.adjustOxyLoss(10, FALSE)
    . = 1
    ..()

**Comment by KorPhaeron** [comment]
kevinz000: also if it wasnt clear my pr is not to be merged because its a meme i made for iteq

**Comment by kevinz000** [comment]
@KorPhaeron PLS NOT EVEN RUNNING IT RUN ITS COURSE AND HAVE A HUGE BALANCE DISCUSSION 👌

**Comment by PKPenguin321** [comment]
To be honest, this isn't *super* powerful considering everybody has epinephrine epipens on them at all times. Basically like cryo sting but stronger

**Comment by dannno** [comment]
haha so fahnny

**Pull request closed without merging**